Add tutorials (#3)
imotions/R-library: imotionsApi/vignettes/article1-example-notebook-explained.Rmd

```{r, eval=F, echo=T}
# Creates metadata for appropriate grouping of the signal
group <- c("", "Skin Temperature")
metadata <- data.table("Provided By" = "iMotions", "Group" = group)

# Uploading the converted signals
ul_temp_signal <- uploadSensorData(params, study, Temp_signal, respondent, "Absolute Skin Temperature",
                                   scriptName = "Skin-Temperature", metadata)
```

After processing, upload either signals (with a Timestamp column) or metrics
(single values, with no Timestamp) to iMotions by calling the function `uploadSensorData`
provided by `imotionsApi`. The data then can be viewed and exported locally through the iMotions Desktop software.

## 8. Generating report

When the notebook is knitted, an HTML file (report) will be generated with specific values for parameters that the user
specified when creating analysis in iMotions. The report normally includes study/analysis information, graphs/tables, algorithm
descriptions and embedded R codes. In this example, the first part specifies study related information
(study, respondent, sensor etc.) and is followed by algorithm descriptions.

## 9. Dropping the R file in the right path

> The properly functioning .Rmd file needs to be put in the path `C:\Users\[username]\Documents\iMotions\RNotebooks` or for export files `C:\Users\[username]\Documents\iMotions\RNotebooks\Exports`, in order to be able to run in iMotions desktop. You may need to create the files called `RNotebooks` or `Exports` yourself for the first time. Please be aware that there could be repeated notebooks of the same kind co-existing, after your editting. You can either delete the repeated ones or keep them separate by using different notebook names.

Reviewer: ```suggestion
The properly functioning .Rmd file needs to be put in the path `C:\Users\[username]\Documents\iMotions\RNotebooks` or for export files `C:\Users\[username]\Documents\iMotions\RNotebooks\Exports`, in order to be able to run in iMotions desktop. You may need to create the folders called `RNotebooks` or `Exports` yourself if they don't exist. Please be aware that there could be repeated notebooks of the same kind co-existing, after your editting. You can either delete the repeated ones or keep them separate by using different notebook names.
```
